### ISSUE TITLE
Eating animated-each's whitespace

### ISSUE THREAD

**Issue by samselikoff** [ember-animation/ember-animated#92]
We're working on using `animated-each` with an array of lines of text that we're rendering inside of a `<pre>`. It looks like this:

```hbs
<AnimatedContainer>
  <pre>
    {{#animated-each exampleTemplateLines key='id' use=codeTransition as |line|~}}
      <div>{{{line.text}}}</div>
    {{~/animated-each}}
  </pre>
</AnimatedContainer>
```

This almost works, but `animated-each` generates some whitespace of its own which breaks our formatted code.

If we make this change to `animated-each`'s template, things work:

```diff
  {{#each renderedChildren key="id" as |child|}}
    {{#-ea-list-element child=child elementToChild=_elementToChild}}
-     {{yield child.value child.index}}
+     {{~yield child.value child.index~}}
    {{/-ea-list-element}}
  {{/each}}
```

What do you think about this? Is this something that would break other uses? Is there another way to solve the problem?

**Comment by ef4** [comment]
This is a good idea. All the components in liquid-fire already do this, we
should do it here too.

**Comment by balinterdi** [comment]
Implemented in #97

**Comment by samselikoff** [comment]
https://github.com/ember-animation/ember-animated/pull/97 closes
